compiler/x/codegen/X86Ops.hpp

```diff
@@ -414,6 +423,11 @@ class TR_X86OpCode
       // OpCodes beyond FENCE are not X86 instructions and hence they do not have binary encodings.
       return op < FENCE ? _binaries[op].encode<TBuffer>(cursor, rex) : cursor;
       }
+   static inline void finalize(TR_X86OpCodes op, uint8_t* cursor)
+      {
+      if (op < FENCE)
```

Comment:
My apologies for letting this languish.  My comment wasn't about the need to finalize an instruction.  Rather, its about how you decide on whether to finalize based on its relative position to another opcode in the table.  Its about coming up with a property to answer the real question you're asking: is this an instruction that has a real binary encoding?   The `_properties` tables are visible to this function, and a property like `PseudoOp` is close to what you want on the current opcodes that currently live after `FENCE` in that enum.  Maybe you need a `NotEncodable` property on the 18 special instructions in that enum, or repurpose the little-used `PseudoOp` property.